runtime/src/lib.rs

```diff
@@ -280,10 +324,20 @@ impl pallet_timestamp::Config for Runtime {
 }
 
 parameter_types! {
-    pub const ExistentialDeposit: u128 = 1 * MILLIUNIT;
+	pub const UncleGenerations: u32 = 0;
```

Comment:
how is a value of 0 better than a value of 5 in 'master' branch?

Reply:
I picked it from https://github.com/paritytech/polkadot/blob/v0.9.13/runtime/polkadot/src/lib.rs#L371 I made it similar to master